src/main/kotlin/three/consulting/epoc/controller/TimesheetEntryController.kt

```diff
@@ -35,6 +35,9 @@ class TimesheetEntryController(private val timesheetEntryService: TimesheetEntry
     fun exportTimesheetEntriesAsCsv(
         response: HttpServletResponse,
         @RequestParam email: String? = null,
+        @RequestParam project: String? = null,
+        @RequestParam customer: String? = null,
+        @RequestParam task: String? = null,
```

Comment:
It's a little ambiguous here what these new parameters are, since these tree are custom types. Looking at the query method itself better names would be `projectName`, `customerName` and `taskName`.